pkg/orchestrator/batch.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package orchestrator
+
+import (
+	"github.com/pingcap/errors"
+	"go.etcd.io/etcd/clientv3"
+)
+
+const (
+	maxBatchPatchSize    = 64
+	maxBatchResponseSize = 64
+)
+
+// getBatchPatches batch patches
+func getBatchPatches(patchGroups [][]DataPatch) ([]DataPatch, int) {
```

Comment:
We also need to count bytes here.  https://etcd.io/docs/v3.3/dev-guide/limit/#:~:text=etcd%20is%20designed%20to%20handle,bytes%20flag%20for%20etcd%20server.